caddytls/config.go

```diff
@@ -528,15 +528,15 @@ var defaultCiphers = []uint16{
 	tls.TLS_RSA_WITH_AES_128_CBC_SHA,
 }
 
-const (
-	// HTTPChallengePort is the officially designated port for
-	// the HTTP challenge.
+var (
+	// HTTPChallengePort must be externally visible as port 80.
 	HTTPChallengePort = "80"
 
-	// TLSSNIChallengePort is the officially designated port for
-	// the TLS-SNI challenge.
+	// TLSSNIChallengePort must be externally visible as port 443.
 	TLSSNIChallengePort = "443"
+)
```

Comment:
I think changing these into vars is not a good idea, these are consts because they are what the ACME spec requires.